### ISSUE TITLE
Effect of parallel sentence classification loss

### ISSUE THREAD

**Issue by thespectrewithin** [facebookresearch/XLM#100]
Dear authors,

Thanks for the great work. I'm wondering if adding additional losses like the parallel sentence classification loss during pretraining (i think its like the next sentence prediction loss from bert but its predicting parallel sentences, right?) will benefit downstream tasks like XNLI? What about the generation/MT loss? Since these losses have been implemented in your codes, would you be able to shed any light on this? Thank you very much! @glample @aconneau

**Comment by glample** [comment]
On XNLI, the implemented classification loss gave improvements over a regular model trained with MLM only. But MLM + TLM is what gave the best performance (see the paper for exact numbers). For MT, we have not tried anything else than MLM only, but it's possible that TLM also helps for low-resource languages.

**Comment by thespectrewithin** [comment]
Thanks! So if my understanding is correct, on XNLI, MLM + TLM is better than MLM + TLM + PC loss, right?

**Comment by glample** [comment]
No, MLM + TLM is better than MLM + PC. Never tried MLM + TLM + PC

**Comment by thespectrewithin** [comment]
Gotcha. Thanks!
